src/Std/Data/DHashMap/Lemmas.lean

```diff
@@ -1512,6 +1512,94 @@ end Const
 
 end monadic
 
+theorem any_eq_toList_any {p : (a : α) → β a → Bool} :
+    m.any p = m.toList.any (fun x => p x.1 x.2) := Raw₀.any_eq_toList_any ⟨m.1, m.2.size_buckets_pos⟩
+
+theorem any_eq_true_iff_exists_mem_get [LawfulBEq α] {p : (a : α) → β a → Bool} :
+    m.any p = true ↔ ∃ (a : α) (h : a ∈ m), p a (m.get a h) :=
+  Raw₀.any_eq_true ⟨m.1, m.2.size_buckets_pos⟩ m.2
+
+theorem any_eq_false_iff_forall_mem_get [LawfulBEq α] {p : (a : α) → β a → Bool} :
+    m.any p = false ↔ ∀ (a : α) (h : a ∈ m), p a (m.get a h) = false :=
+  Raw₀.any_eq_false ⟨m.1, m.2.size_buckets_pos⟩ m.2
```

Comment:
Would it make sense to name these just `any_eq_true` and `any_eq_false`, like the corresponding `Array` lemmas, and similarly for the `all` variants?

Reply:
Okay, that makes sense. Seems like the situation for maps is slightly different compared to `Array`. No change necessary in my opinion.